### PR TITLE
BUG: add missing libpng dependency to g2lib-1.4.0 for intel-2017a

### PR DESCRIPTION
**Pull request by andreas-h**
When trying to build g2lib-1.4.0-intel-2017a, I got an error about `png.h` not being found. With this fix, the build succeeds.

### CODE REVIEW

**Comment by andreas-h** [comment]
The same was true for g2clib

File: easybuild/easyconfigs/g/g2clib/g2clib-1.6.0-intel-2017a.eb [hunk 1]
```diff
@@ -9,7 +9,8 @@ toolchain = {'name': 'intel', 'version': '2017a'}
 source_urls = [homepage]
 sources = ['%(name)s-%(version)s.tar']
 
-dependencies = [('JasPer', '1.900.1')]
+dependencies = [('JasPer', '1.900.1'),
+                ('libpng', '1.6.29'),]
```
Comment (boegel) [inline easybuild/easyconfigs/g/g2clib/g2clib-1.6.0-intel-2017a.eb]: @andreas-h Please reformat to match style used in other easyconfigs:

```python
dependencies = [
    ('JasPer', '1.900.1'),
    ('libpng', '1.6.29'),
]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2670.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4ee6ea46501fa64918ba6276f7436219 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2127.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/73e7b777d097e1b8ef910830619034cb for a full test report.

**Comment by boegel** [comment]
Going in, thanks @andreas-h!

**Pull request merged**